e2e/test/scenarios/visualizations-tabular/table.cy.spec.js

```diff
@@ -216,7 +216,7 @@ describe("scenarios > visualizations > table", () => {
         },
       ],
     ].forEach(([column, test]) => {
-      cy.get(".cellData").contains(column).trigger("mouseover");
```

Comment:
Any reason we don't use `cy.findByTestId`? I don't mind either way but I'm curious

Reply:
Ok, no worries if it's causing you issues